tests/compatibility/SyncOperationsTestCase.test.nut

```diff
@@ -0,0 +1,97 @@
+// MIT License
+//
+// Copyright 2017 Electric Imp
+//
+// SPDX-License-Identifier: MIT
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO
+// EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES
+// OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+// ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+// OTHER DEALINGS IN THE SOFTWARE.
+// "Promise" symbol is injected dependency from ImpUnit_Promise module,
+// while class being tested can be accessed from global scope as "::Promise".
+
+@include __PATH__+"/../Core.nut"
+
+// ReadOneSectorTestCase
+// Tests for SPIFlashLogger.read()
+class SyncOperationsTestCase extends Core {
+
+    logger = null;
+    _postfix = " - 09876543210987654321098765432109876543210987654321";
+    _max_sector_logs = 62;
+
+    function setUp() {
+        return Promise(function(resolve, reject) {
+            try {
+                if (!isAvailable()) {
+                    resolve();
+                    return;
+                }
+                local start = 0;
+                local end = start + 2;
+                start *= SPIFLASHLOGGER_SECTOR_SIZE;
+                end   *= SPIFLASHLOGGER_SECTOR_SIZE;
+                logger = SPIFlashLogger(start, end);
+                // Erase all data
+                logger.eraseAll(true);
+
+                for (local i = 0; i <= _max_sector_logs; i++) { // 175
```

Comment:
// 175 ?

Reply:
Dropped, thanks  (it is total number of objects at this place)